src/components/screens/Settings/SettingsIndexScreen.tsx

```diff
@@ -186,6 +206,12 @@ function SettingsIndexScreen({
               onCacheClear();
             }}
           />
+          <CCell
+            cellStyle="Basic"
+            title={t("Report a Bug")}
```

Comment:
I think saying "Report a Bug on Github" or something, because it makes you log into GitHub and i think a lot of users who don't use github will be confused and ask us to allow reporting bugs without it or something.

Reply:
Good point

Reply:
done